### PR TITLE
Read default values from file at /etc/openssh-github-keys

### PR DESCRIPTION
**Pull request by sestrella**

### CODE REVIEW

**Comment by sestrella** [comment]
@jsl, @jpvillaisaza could you guys take a look please

File: spec/System/OpensshGithubKeys/ConfigSpec.hs [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    it "uses the default dotfile" $ do
+      let defaults = [("dotfile", "/path/to/env")]
+      let args = [ "--organization=stackbuilders"
+                 , "--team=team"
+                 , "--user=user-to-add"
+                 , "user"
+                 ]
+      getParseResult (execParserPure parserPrefs (options defaults) args) `shouldBe` Just opts
+
+    it "overwrites the default dotfile" $ do
```
Comment (jsl) [inline spec/System/OpensshGithubKeys/ConfigSpec.hs]: overrides

File: src/Main.hs [hunk 2]
```diff
@@ -42,44 +29,13 @@ fetchKeys opts = do
 
     else return ()
```
Comment (jpvillaisaza) [inline src/Main.hs]: This `if any... then do... else return ()` can be replaced by `when any... (do...)`.

**Comment by sestrella** [comment]
@jpvillaisaza thanks for your comment, I just pushed some changes, do you mind take a second look please

**Comment by jpvillaisaza** [comment]
@sestrella Looks good, though I'd probably had to take a closer look to get a better idea of all the code. For .gitignore files, I usually just copy the appropriate one from https://github.com/github/gitignore, Haskell.gitignore in this case.

File: src/System/OpensshGithubKeys/Config.hs [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+optionsParser :: [(String, String)] -> Parser Options
+optionsParser v = Options <$> authenticateArgument
+                  <*> organizationOption (lookup "organization" v)
+                  <*> teamOption (lookup "team" v)
+                  <*> usersOption
+                  <*> dotfileOption (lookup "dotfile" v)
+
+authenticateArgument :: Parser String
+authenticateArgument = argument str $ metavar "AUTHENTICATE" <> help "Local user trying to authenticate currently"
```
Comment (jsl) [inline src/System/OpensshGithubKeys/Config.hs:43]: Can you change to, "Local user currently trying to authenticate"?

**Pull request merged**